src/main/webapp/WEB-INF/themes/mobile/resource/css/peer-review.css

```diff
@@ -0,0 +1,110 @@
+#peer-review {
+  margin: 0;
+}
+
+.page-title,
+.section-title {
+  font-size: 24px;
+  line-height: 36px;
+  font-weight: 400;
+  margin: 12px 0 24px;
+  font-family: Arial, sans-serif;
+}
+
+.page-title {
+  margin-left: 1rem;
+}
+.review-history {
+  border-collapse: collapse;
+  border-spacing: 0;
+  border-top: 1px solid #eee;
+  width: 100%;
+  margin: 0;
+  text-align: left;
+}
+.review-history,
+.supplementary-material,
+.tpr-info {
+  font-size: 14px;
+  line-height: 18px;
+}
+
+.decision-letter,
+.author-response,
+.tpr-info {
+  padding: 12px 0;
+  margin: 0 0 0 1rem;
+}
+
+.decision-letter__body,
+.author-response {
+  color: #999;
+}
+
+.decision-letter__label {
+  display: block;
+  width: 100%;
+}
+
+.review-history p,
+.author-response p,
+.tpr-info p,
+.review-files__title,
+.supplementary-material__caption {
+  margin: 12px 0;
+}
+
+.review-files__title {
+  font-weight: 600;
+}
+
+.supplementary-material__caption {
+  margin: 0 0 24px;
+}
+.supplementary-material__label {
+  font-weight: 600;
+  text-transform: uppercase;
+  padding: .5em 1em;
+  text-align: center;
+  display: inline-block;
+  line-height: 24px;
+  -webkit-border-top-right-radius: 3px;
+  -webkit-border-bottom-right-radius: 3px;
+  -webkit-border-bottom-left-radius: 3px;
+  -webkit-border-top-left-radius: 3px;
+  -moz-border-radius-topright: 3px;
+  -moz-border-radius-bottomright: 3px;
+  -moz-border-radius-bottomleft: 3px;
+  -moz-border-radius-topleft: 3px;
+  border-top-right-radius: 3px;
+  border-bottom-right-radius: 3px;
+  border-bottom-left-radius: 3px;
+  border-top-left-radius: 3px;
+}
+
+.review-history tr {
+  border-bottom: 1px solid #eee;
+}
+
+.version {
+  background: #f9f9f9;
+  padding: 12px 0;
+}
+
+.decision-letter__date,
+.decision__date {
+  width: 100%;
+  display: block;
+}
+
+.decision__date {
+  font-weight: 400;
+  text-indent: 1rem;
+}
+
+.decision-letter__title {
```

Comment:
@samV-plos I expect there to exist analogous `author-comment...` and `acceptance-letter...` classes. How do you want to handle the css classes so that they are all styled the same as `decision-letter...`?

Reply:
yes, we should normalize these. 
I realized that some of the content was keying off the classes in the mark-up, so wasn't sure if they were ok to change.